.jscsrc.json

```diff
@@ -1,5 +1,6 @@
 {
   "preset": "google",
+  "esnext": true,
```

Comment:
update jscs and this is included by default (the options has been removed in 3.0)